uniprot-search/src/main/java/org/uniprot/store/search/field/SearchFields.java

```diff
@@ -13,29 +15,29 @@
 public interface SearchFields {
```

Comment:
Can we not make this interface an abstract class? I am saying this because i see many default methods.

Reply:
unfortunately, since its usage is via an enum, this is the only choice (enums can only implement interfaces).

Reply:
I agree with you.